lib/src/http/managers/message_manager.dart

```diff
@@ -475,6 +486,14 @@ class MessageManager extends Manager<Message> {
     return SubmittedTextDisplayComponent(id: raw['id'] as int);
   }
 
+  SubmittedFileUploadComponent parseSubmittedFileUploadComponent(
+      Map<String, Object?> raw) {
+    return SubmittedFileUploadComponent(
+        id: raw['id'] as int,
+        customId: raw['custom_id'] as String,
+        values: parseMany(raw['values'] as List));
```

Comment:
Missing a `Snowflake.parse` argument here. A quick serialisation test would've caught this.